### PR TITLE
Integrate messaging permission switch into the meeting creation view.

### PR DESCRIPTION
**Pull request by maheichyk**
Adds a messaging permission switch into the meeting creation view.

Screenshot 2023-07-03 at 15 34 56

**:heavy_check_mark: Checklist**

- [x] A changeset describing the change and affected packages ([more info](https://github.com/nordeck/.github/blob/main/docs/CONTRIBUTING.md#changelog-and-versioning)).
- [ ] Added or updated documentation.
- [x] Tests for new functionality and regression tests for bug fixes.
- [x] Screenshots or videos attached (for UI changes).
- [x] All your commits have a `Signed-off-by` line in the message ([more info](https://github.com/nordeck/.github/blob/main/docs/CONTRIBUTING.md#dco)).

### CODE REVIEW

**Comment by nils-nord** [comment]
Test passed. 

Tested using Google Chrome, Mozilla Firefox and Microsoft Edge.

**Review by ahmadkadri — DISMISSED** [review]

**Review by ahmadkadri — DISMISSED** [review]

**Review by nurjinjafar — DISMISSED** [review]
looks good to me!

**Review by ahmadkadri — APPROVED** [review]

File: matrix-meetings-bot/src/dto/MeetingCreateDto.ts [hunk 1]
```diff
@@ -101,6 +103,14 @@ export class MeetingCreateDto {
   @IsOptional()
   participants?: ParticipantDto[];
 
+  /**
+   * power level required to send events in the room
+   */
+  @IsNumber({ maxDecimalPlaces: 0 })
+  @Min(0)
+  @IsOptional()
+  messaging_power_level?: number;
```
Comment (dhenneke) [inline matrix-meetings-bot/src/dto/MeetingCreateDto.ts:112]: We should also update the API definition in the https://github.com/nordeck/matrix-meetings/blob/main/docs/data-model.md file. Both for the event schema and the REST API endpoint.

Comment (maheichyk) [reply matrix-meetings-bot/src/dto/MeetingCreateDto.ts:112]: thanks for finding, this PR updates the API: https://github.com/nordeck/matrix-meetings/pull/201

**Pull request merged**